### PR TITLE
Add 6 new values to the 'cisco_ios_show_ip_bgp_summary.textfsm' template

### PR DESCRIPTION
**Pull request by PavloSkliarenko**
Added 6 new values to the template, which were previously skipped as unnecessary:
 - NEIGH_V
 - MSG_RCVD
 - MSG_SENT
 - TB_VER
 - IN_Q
 - OUT_Q

### CODE REVIEW

File: ntc_templates/templates/cisco_ios_show_ip_bgp_summary.textfsm [hunk 1]
```diff
@@ -2,16 +2,22 @@ Value Filldown,Required ROUTER_ID ([0-9a-f:\.]+)
 Value Filldown LOCAL_AS (\d+(\.\d+)?)
 Value Filldown ADDR_FAMILY (.+?)
 Value BGP_NEIGH (\d+?\.\d+?\.\d+?\.\d+?)
+Value NEIGH_V (\d+)
 Value NEIGH_AS (\d+)
+Value MSG_RCVD (\d+)
```
Comment (jvanderaa) [inline ntc_templates/templates/cisco_ios_show_ip_bgp_summary.textfsm:7]:
```suggestion
Value MESSAGES_RECIEVED (\d+)
```

File: ntc_templates/templates/cisco_ios_show_ip_bgp_summary.textfsm [hunk 2]
```diff
@@ -2,16 +2,22 @@ Value Filldown,Required ROUTER_ID ([0-9a-f:\.]+)
...
+Value MSG_SENT (\d+)
+Value TB_VER (\d+)
+Value IN_Q (\d+)
+Value OUT_Q (\d+)
```
Comment (jvanderaa) [inline ntc_templates/templates/cisco_ios_show_ip_bgp_summary.textfsm:11]:
```suggestion
Value MESSAGES_SENT (\d+)
Value TABLE_VER (\d+)
Value INPUT_QUEUE (\d+)
Value OUTPUT_QUEUE (\d+)
```

**Review by jvanderaa — CHANGES_REQUESTED** [review]
We are working on a data model normalization with more complete keys. I'd like to start with this PR, and if you want to also expand some of the other keys that were not being added for this PR, I would very much appreciate it. Major release coming within a week or so.

**Comment by PavloSkliarenko** [comment]
Is now everything alright?

**Comment by PavloSkliarenko** [comment]
>

**Pull request closed without merging**